sdks/python/apache_beam/io/gcsio.py

```diff
@@ -47,7 +47,7 @@
       'Google Cloud Storage I/O not supported for this execution environment '
       '(could not import storage API client).')
 
-DEFAULT_READ_BUFFER_SIZE = 1024 * 1024
+DEFAULT_READ_BUFFER_SIZE = 16 * 1024 * 1024
```

Comment:
As mentioned, please place a dated summery of these benchmarks here, or a pointer to somewhere more permanent.